packages/forms/src/Components/Select.php

```diff
@@ -395,7 +395,10 @@ public function getSearchResults(string $search): array
             $results = $results->toArray();
         }
 
-        return $results;
+        return collect($results)
+            ->map(fn ($option, $key) => [$key => $option])
+            ->values()
+            ->all();
```

Comment:
This should be probably moved to a higher level as people might overwrite `getSearchResults()`?

Reply:
Done